baybe/simulation/scenarios.py

```diff
@@ -5,26 +5,18 @@
 from collections.abc import Callable
 from copy import deepcopy
 from dataclasses import dataclass
-from typing import Any, Literal
+from typing import TYPE_CHECKING, Any, Literal
 
 import numpy as np
 import pandas as pd
 
-try:
-    import xyzpy as xyz
-    from xarray import DataArray
-except ImportError as ex:
-    # This is just to augment the error message and provide a suggestion
-    raise ModuleNotFoundError(
-        "xyzpy is not installed, the simulation module is unavailable. Consider "
-        "installing baybe with `simulation` dependency, e.g.`pip install "
-        "baybe[simulation]`"
-    ) from ex
-
 from baybe.campaign import Campaign
 from baybe.exceptions import NothingToSimulateError
 from baybe.simulation.core import simulate_experiment
 
+if TYPE_CHECKING:
+    from xarray import DataArray
```

Comment:
can we get rid of the `xyz` package again? it has somehow bothered me that its adds a dependency for simulations. Is it adding something otherwise not possible? Speedup? Or just convenience? If its just the latter we could kick it... (not this PR)

Reply:
A mix of convenience and enabling something otherwise not possible:
* Convenient, because it allows us to easily sweep over different simulation parameters. Of course, you *could* to this without the package, by writing you own looping mechanism etc. But at this point, the question is very much the same like "do you need attrs" --> of course not, but .... you know where this is going. 
* Enabling, because we can easily switch on, for example, parallel simulation. And for that, you would otherwise probably use something like joblib as well, so again another dependency.

I understand your concern though, and I'm not saying that we must necessarily keep it. But since I think it's currently very much at the lower end of the priority list, let me phrase it that way: feel free to create a PR and see, if the removal is justified or not (i.e. if we get 200 lines of extra code, probably it's better to keep it 😄)

Reply:
i think what attrs provides alltogether seems much heavier than this package (just doing parallel loops right?) so getting rid of it still seems reasonable to me
but also agree on low prio